### ISSUE TITLE
Docker install doc has old invocation of calicoctl

### ISSUE THREAD

**Issue by lwr20** [projectcalico/calico#359]
http://docs.projectcalico.org/v2.0/getting-started/docker/installation/gce suggests that you should add a profile like this: 
```
$ calicoctl profile add WEB
$ calicoctl profile WEB rule show
```

This no longer works.  There's a bunch more like this in that doc too.

**Comment by lwr20** [comment]
Similarly in 
http://docs.projectcalico.org/v2.0/getting-started/docker/installation/digital-ocean
http://docs.projectcalico.org/v2.0/getting-started/docker/installation/aws

**Comment by robbrockbank** [comment]
@turk has a PR up to remove those docs for now.

**Comment by djosborne** [comment]
Yep, Fixed by #331 (as those guides are all removed)
